frontend/src/pages/roles/Roles.tsx

```diff
@@ -53,85 +50,65 @@ function Roles() {
-        const updatedRole: RoleDTO = { ...roles[activeRole], permissions: newPermissions };
-        setRoles({ ...roles, [updatedRole.id]: updatedRole });
-    };
-
-    const handleSaveRole = async () => {
-        await userContext.fetcher(
-            'PATCH /api/roles/:roleId',
-            {
-                permissions: roles[activeRole].permissions,
-            },
-            { roleId: roles[activeRole].id },
-        );
-    };
-
     return (
-        <div className=" md:pl-[90px] bg-white overflow-scroll h-screen md:ml-4">
-            <div className="px-4 pt-2 flex flex-row h-[5%]">
-                <Button size="medium" color="#262a6c" onClick={returntoProfile}>
-                    <IonIcon name="chevron-back-outline" /> Back{' '}
-                </Button>
+        <div className=" md:pl-[90px] bg-white mx-2 h-screen flex flex-col max-h-screen min-h-0">
+            <div className="my-2 flex flex-row">
+                <span
+                    onClick={returntoProfile}
+                    className="flex flex-row items-center px-2 py-1 pr-3 cursor-pointer border-slate-300 border-solid border-[1px] rounded-md"
+                >
+                    <IonIcon name="chevron-back-outline" />
+                    Back
+                </span>
             </div>
-            <div className=" w-full p-4 rounded-md md:grid md:grid-cols-3 md:gap-12 overflow-scroll h-full">
-                <div className="flex flex-col">
-                    <div className="h-[40%]">
-                        <RoleList
-                            roles={Object.values(roles)}
-                            handleChangeActiveRole={setActiveRole}
-                            handleAddRole={handleAddRole}
-                        />
-                    </div>
-                    <div className="h-[40%]">
-                        <UserList users={users} />
-                    </div>
-                </div>
-
-                <div className="col-span-2 p-2 rounded-md mt-10 md:mt-0 h-fit">
-                    {activeRole ? (
-                        <>
-                            <PermissionsList
-                                activeRole={roles[activeRole].name}
-                                permissions={roles[activeRole].permissions}
-                                setPermissions={handleSetPermissions}
+            <TabManager
+                orientation="row"
+                content={[
+                    { tabTitle: 'Wowzers', tabData: ['role1', 'role2'] },
+                    { tabTitle: 'Roles' },
+                    { tabTitle: 'Users', tabData: ['user1', 'user2'] },
+                ]}
+                loader={(data) => {
+                    if (data.tabTitle === 'Wowzers') {
+                        return (
+                            <TabManager
+                                orientation="column"
+                                content={[
+                                    { tabTitle: 'Thexboss', tabData: ['Awesome', '🥚'] },
```

Comment:
LMFAO, lets get rid of this ahhahahahhaha

Reply:
Done